tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -95,7 +96,6 @@ def test_zos_job_id_query_multi_wildcards_func(ansible_zos_module):
                 assert qresult.get("jobs") is not None
                 assert qresult.get("jobs")[0].get("system") is not None
                 assert qresult.get("jobs")[0].get("subsystem") is not None
-                assert qresult.get("jobs")[0].get("execution_time") is not None
```

Comment:
Same comment as below, we should keep the assertions in `zos_job_query` response too.